package.json

```
@@ -59,7 +59,7 @@
     "lodash": "^4.17.11",
     "numeral": "^2.0.6",
     "prop-types": "^15.6.0",
-    "react-ace": "^7.0.5",
+    "react-ace": "8.0.0",
```

Let's leave the same range
```suggestion
    "react-ace": "^8.0.0",
```